Libraries/LibJS/Runtime/MathObject.cpp

```diff
@@ -125,10 +125,13 @@ JS_DEFINE_NATIVE_FUNCTION(MathObject::floor)
 JS_DEFINE_NATIVE_FUNCTION(MathObject::ceil)
 {
     auto number = interpreter.argument(0).to_number(interpreter);
+    auto number_double = number.as_double();
```

Comment:
This needs to be after the exception check below as `to_number()` can return an empty value (if it throws), and `as_double()` would assert in that case.